### PR TITLE
feat: Introducing GridLossResultProducedV1 event (without publishing it yet)

### PR DESCRIPTION
**Pull request by JesperGomes**
Adding new event intended to be used for publishing grid loss results (positive and negative grid loss). It is not used for anything yet (not hooked up to handler)

### CODE REVIEW

File: source/dotnet/wholesale-api/Events/Events.Infrastructure/IntegrationEvents/GridLossResultProducedV1/Factories/GridLossResultProducedV1Factory.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+using Energinet.DataHub.Wholesale.Events.Infrastructure.IntegrationEvents.GridLossResultProducedV1.Mappers;
+using Google.Protobuf.WellKnownTypes;
+
+namespace Energinet.DataHub.Wholesale.Events.Infrastructure.IntegrationEvents.GridLossResultProducedV1.Factories;
+
+public class GridLossResultProducedV1Factory : IGridLossResultProducedV1Factory
+{
+    public Contracts.IntegrationEvents.GridLossResultProducedV1 Create(EnergyResult result)
+    {
+        var energyResultProduced = new Contracts.IntegrationEvents.GridLossResultProducedV1
```
Comment (energinet-ajw) [inline source/dotnet/wholesale-api/Events/Events.Infrastructure/IntegrationEvents/GridLossResultProducedV1/Factories/GridLossResultProducedV1Factory.cs:27]: Should be gridLossResultProducedV1.

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/EnergyResults/EnergyResult.cs [hunk 2]
```diff
@@ -41,6 +41,7 @@ public sealed class EnergyResult : AggregatedTimeSeries
         PeriodStart = periodStart;
         PeriodEnd = periodEnd;
         FromGridArea = fromGridArea;
+        MeteringPointId = string.Empty; // TODO: use constructor parameter instead
```
Comment (energinet-ajw) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/EnergyResults/EnergyResult.cs:44]: TODO?

File: source/dotnet/wholesale-api/Events/Events.Infrastructure/IntegrationEvents/GridLossResultProducedV1/GridLossResultProducedV1.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+namespace Energinet.DataHub.Wholesale.Contracts.IntegrationEvents;
+
+public partial class GridLossResultProducedV1 : IEventMessage
+{
+    /// <summary>
+    /// The message type for transport message meta data in accordance with ADR-008.
+    /// </summary>
+    public const string EventName = "GridLossResultProducedV1";
+
+    public const int EventMinorVersion = 1;
```
Comment (energinet-ajw) [inline source/dotnet/wholesale-api/Events/Events.Infrastructure/IntegrationEvents/GridLossResultProducedV1/GridLossResultProducedV1.cs:27]: Starter minor med 0?

File: source/dotnet/wholesale-api/Events/Events.Infrastructure/IntegrationEvents/GridLossResultProducedV1/Protos/grid_loss_result_produced_v1.proto [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+     * Example: 65801e3c-5131-426e-b1bb-7b186349c996
+     */
+    string calculation_id = 1;
+
+    /*
+     * The beginning of the calculation period.
+     * The time is included in the period and is identical to the time of
+     * the first time series point in the results.
+     */
+    google.protobuf.Timestamp period_start_utc = 3;
```
Comment (energinet-ajw) [inline source/dotnet/wholesale-api/Events/Events.Infrastructure/IntegrationEvents/GridLossResultProducedV1/Protos/grid_loss_result_produced_v1.proto:40]: Why not 2?

File: source/dotnet/wholesale-api/Events/Events.UnitTests/Infrastructure/IntegrationEvents/GridLossResultProducedV1/GridLossResultsProducedV1Tests.cs [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+using Xunit;
+
+namespace Energinet.DataHub.Wholesale.Events.UnitTests.Infrastructure.IntegrationEvents.GridLossResultProducedV1;
+
+public class GridLossResultsProducedV1Tests
+{
+    [Fact]
+    public void LargestCalculationResult_FitsInServiceBusMessage()
+    {
+        var largestRsm014Result = new Contracts.IntegrationEvents.GridLossResultProducedV1
```
Comment (energinet-ajw) [inline source/dotnet/wholesale-api/Events/Events.UnitTests/Infrastructure/IntegrationEvents/GridLossResultProducedV1/GridLossResultsProducedV1Tests.cs:27]: Missing ACT and ARRANGE.

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/EnergyResults/EnergyResult.cs (hunk 2 above)
Comment (JesperGomes) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/EnergyResults/EnergyResult.cs:44]: Yes, it's waiting for the data to be available in delta table.

**Review by energinet-ajw — APPROVED** [review]

**Pull request merged**